lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -63,7 +63,7 @@ module AuthenticatableAutosaveExtensions
       module ClassMethods
         def find_first_by_auth_conditions(tainted_conditions, opts = {})
           filtered_conditions = devise_parameter_filter.filter(tainted_conditions.dup)
-          criteria = filtered_conditions.extract!(:email, :unconfirmed_email)
+          criteria = filtered_conditions.extract!(:email, :unconfirmed_email, :confirmation_token)
```

Comment:
Could you explain what's this new option for?  Sorry it's been a while and I have actually forgotten some of the options.

Reply:
This is for people who've received the confirmation email, which includes a confirmation token. We need to be able to find the email record also via this confirmation token.

Reply:
Nice catch, thanks!